tests/functional/aws-node-sdk/test/multipleBackend/get.js

```diff
@@ -106,6 +113,15 @@ describe('Multiple backend get object', () => {
                         done();
                     });
             });
+            it('should get an object from AWS', done => {
```

Comment:
please also add test getting 0 byte object.  also, need to test that you are getting the user metadata back correctly in both this test and the 0 byte object test.